mempool/mempool.go

```diff
@@ -482,7 +481,7 @@ func (mem *Mempool) ReapMaxBytesMaxGas(maxBytes, maxGas int64) types.Txs {
 	for e := mem.txs.Front(); e != nil; e = e.Next() {
 		memTx := e.Value.(*mempoolTx)
 		// Check total size requirement
-		aminoOverhead := int64(amino.UvarintSize(uint64(len(memTx.tx))))
+		aminoOverhead := types.ComputeAminoOverhead(memTx.tx, 1)
```

Comment:
https://github.com/tendermint/tendermint/pull/2792/files#r232387921

Reply:
nevermind - we want to compute the overhead for when it appears in the block, not what it might look like in the mempool